stories/table-chart.stories.js

```diff
@@ -0,0 +1,202 @@
+*/
+
+const compactStyleClasses = makeStyles(({
+  tableRootContainer: {
+    '& .table__container': {
+      '& .table__content-container': {
+        '& .table__header-compact': {
+          '& .table__header-row': {
+            '& .table__header-cell': {
+              padding: '0.25rem 0 0.25rem 0.5rem',
+              '&:last-child': {
+                paddingRight: '0.5rem',
+              },
+            },
+          },
+        },
+    
+        '& .table__body-compact': {
+          '& .table__body-row': {
+            '& .table__body-cell': {
+              padding: '0.25rem 0 0.25rem 0.5rem',
+              '&:last-child': {
+                paddingRight: '0.5rem',
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+}))
+
+const numberFormatting = val => `${Number(val)/1000}km`
+
+const percentageFormatting = val => `${Number(val).toFixed(2)}%`
+
+export const bar = () => <Table data={provinces} rowsPerPage={[5,10,20,50]} barColumns />
+
+export const barSelectiveColumns = () => <Table data={provinces} rowsPerPage={[5,10,20,50]} barColumns={['rate', 'new_cases']} />
+
+export const dashboardTable = () => (
+  <Table 
+    data={data}
+    columns={
+      Object.keys(data[0]).map((key) => ({
+        Header: key.replaceAll('_', ' '), 
+        accessor: key,
+        disableFilters: true,
+        disableSortBy: key === 'Age' ? false : true,
+      }))
+    }
+    sortBy={{ id: 'Age', desc: false }}
+    rowsPerPage={[5,10,20,50]}
+    barColumns={['Marketshare of Vehicle Age']}
+    hidePagination 
+    formatData={{ 
+      'Resolution': numberFormatting,
+      'Marketshare of Vehicle Age': {
+        func: percentageFormatting,
+        type: '%',
+      },
+    }}
+    toolbar={false}
+    headerTitle={true}
+    title='testing title'
+  />
+)
+
+export const customColorAndBorder = () => (
+  <Table 
+    data={data} 
+    columns={
+      Object.keys(data[0]).map((key) => ({
+        Header: key.replaceAll('_', ' '), 
+        accessor: key,
+        disableFilters: true,
+        disableSortBy: key === 'Age' ? false : true,
+      }))
+    }
+    sortBy={{ id: 'Age', desc: false }}
+    rowsPerPage={[5,10,20,50]}
+    barColumns={['Marketshare of Vehicle Age']}
+    hidePagination 
+    formatData={{ 
+      'Resolution': {
+        func: numberFormatting,
+        type: 'Km',
+      },
+      'Marketshare of Vehicle Age': {
+        func: percentageFormatting,
+        type: '%',
+      },
+    }}
+    toolbar={false}
+    headerTitle={true}
+    title='testing title'
+    barColumnsColor={`${getTailwindConfigColor('success-400')}`} // only HEX color
+    defaultStyles={
+      {
+        headerColor: 'grey', // PropTypes.oneOf(['grey', 'white'])
+        borderType: 'around', // PropTypes.oneOf(['none', 'horizontal', 'vertical', 'around'])
+        centerHeader: true,
+      }
+    }
+  />
+)
+
+export const customCompactStyle = () => (
+  <Table 
+    data={data} 
+    classes={{
```

Comment:
I see that we will have to add this compact styling in widget studio when implementing the table there. How about we add a prop (ex: compact) and save these compact styling in lumen-table and add them in lumen-table if the prop is true?
This way we help users have a compact table without the hustle of thinking how to create this special styling to pass on to the table.